odxtools/odxtypes.py

```diff
@@ -28,7 +28,7 @@ def bytefield_to_bytearray(bytefield: str) -> bytearray:
 # sub-parameters, so this is a recursive type, and fields encompass
 # multiple items, so this can be a list of objects.
 TableStructParameterValue = Tuple[str, "ParameterValue"]
-ParameterValue = Union[AtomicOdxType, "ParameterValueDict", TableStructParameterValue,
+ParameterValue = Union[None, AtomicOdxType, "ParameterValueDict", TableStructParameterValue,
```

Comment:
why?

Reply:
the reference to a structure  is optional for multiplexer cases. I thought that it is kind of intuitive to use `None` as the physical value...

Reply:
alternatively, we could use an empty dict,  but IMO `None` is a bit better...

Reply:
I've [changed my mind](https://github.com/mercedes-benz/odxtools/pull/318/commits/3f0add18bb82d8198c195379725860a75e0c85a9): always returning a dict will remove a "footgun" because the code which processes the decoded message does not need to expect `None`. thanks for bringing this up.

Reply:
it is included in https://github.com/mercedes-benz/odxtools/commit/3f0add18bb82d8198c195379725860a75e0c85a9